custom_slack_provider/views.py

```diff
@@ -76,15 +76,16 @@ def get_data(self, token):
         resp['user']['full_name'] = user_info.get('profile',
                                                   {}).get('real_name')
         resp['user']['first_name'] = user_info.get('profile',
-                                                  {}).get('first_name')
+                                                   {}).get('first_name')
```

Comment:
Is this one indented properly? They rest look 1-space to the left still.

Reply:
I am pretty sure it's correct. It should be in line with `'profile'` which it is now.